cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
 is_a: QC:4000022 ! chromatogram metric
 relationship: has_relation QC:4000014 ! QC metric relation: multiple runs
 
-[Typedef]
-id: has_relation
-name: has_relation
+[Term]
+id: QC:4000107
+name: Column type
+def: "The parent to the group of column definitions for name, content, and units for table type." [PSI:QC]
+relationship: has_relation QC:4000006 ! table
+
+[Term]
+id: QC:4000108
+name: Retention time
+def: "The column contains retention time points in seconds." [PSI:QC]
+is_a: QC:4000107 ! Column type
+property_value: has_units UO:0000010
+
+[Term]
+id: QC:4000109
+name: Relative intensity
+def: "The column contains relative intensity values of one type." [PSI:QC]
```

Comment:
what is meant by `one type`?

Reply:
the unit is a family term with [multiple](https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1000043&viewMode=All&siblings=false), but the column is supposed to only contain  values of one type.